pkg/pkg.bzl

```diff
@@ -415,10 +443,16 @@ pkg_deb_impl = rule(
         "replaces": attr.string_list(default = []),
         "provides": attr.string_list(default = []),
 
-        # Outputs.
+        # Common attributes
         "out": attr.output(mandatory = True),
-        "deb": attr.output(mandatory = True),
-        "changes": attr.output(mandatory = True),
+        "package_file_name": attr.string(doc = "See Common Attributes"),
+        "package_variables": attr.label(
+            doc = "See Common Attributes",
```

Comment:
Could this be a markdown hyperlink? This occurs a few places below (and maybe above, I might have missed some).

Reply:
We need a PR just to make the doc strings usable to generate docs. Right now this is unused, so it is sort of moot.
I am waiting for improvements to Stardoc.